### PR TITLE
DRA: add constant PluginClientTimeout

### PR DESCRIPTION
**Pull request by bart0sh**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

This is a tiny update for the code introduced by https://github.com/kubernetes/kubernetes/pull/114844
It addresses [this comment](https://github.com/kubernetes/kubernetes/pull/114844#discussion_r1102814063)

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

### CODE REVIEW

File: pkg/kubelet/cm/dra/plugin/client.go [hunk 1]
```diff
@@ -32,6 +32,8 @@ import (
 	drapbv1 "k8s.io/kubelet/pkg/apis/dra/v1alpha1"
 )
 
+const PluginTimeout = 10 * time.Second
```
Comment (klueska) [inline pkg/kubelet/cm/dra/plugin/client.go:35]: Can you call this PluginClient timeout

also, I remember there were two places where this constant should be used, not just one

Comment (bart0sh) [reply pkg/kubelet/cm/dra/plugin/client.go]: thank you for the review! done

**Pull request merged**